circle.yml

```diff
@@ -22,6 +22,8 @@ test:
 
   override:
     - git diff --exit-code > $CIRCLE_ARTIFACTS/diff || (echo "Compiled output is not up-to-date" && test $CIRCLE_BRANCH != $MASTER_BRANCH)  # fail build only on main branch
+    - npm run lint:
+        parallel: true
```

Comment:
If you specify `parallel`, you have to specify how files should be split with the `files` key ([example](https://github.com/sgmap/mes-aides-ui/blob/rewrite/circle.yml#L43-L47), [reference](https://circleci.com/docs/parallel-manual-setup)). Otherwise, you're just running the exact same operation on several distinct containers, losing time rather than gaining any.